addon/ng2/blueprints/component/index.js

```diff
@@ -13,7 +14,9 @@ module.exports = {
 
   locals: function(options) {
     return {
-      dynamicPath: this.dynamicPath.dir
+      dynamicPath: this.dynamicPath.dir === path.sep
```

Comment:
this sounds like a `path.resolve(this.dynamicPath.dir)` or something like that, no?